src/Flux/neural_de.jl

```diff
@@ -24,3 +24,24 @@ function neural_dmsde(model,x,mp,tspan,
   # TODO could probably use vcat rather than collect here
   solve(prob, args...; kwargs...) |> Tracker.collect
 end
+
+# Flux Layer Interface
+struct NeuralODE
+    model
+    tspan
+    solver
+    args
+    kwargs
+end
+
+# Optional args and kwargs
+NeuralODE(model,tspan,solver,kwargs)= NeuralODE(model,tspan,solver,(),kwargs)

```

Comment:
This is related to the `args...` question. Is there a better way to allow both `args` and `kwargs` to be optional?

Reply:
The suggestion I did will naturally allow passing both and pull the splats into the appropriate fields